xwiki-platform-core/xwiki-platform-tour/xwiki-platform-tour-ui/src/main/resources/TourCode/TourJS.xml

```diff
@@ -172,7 +172,10 @@ require(['jquery', 'xwiki-meta'], function ($, xm) {
     // Create the popover
     var popover = $('&lt;div class="popover-content"&gt;#tr("tour.popover.show.hint")&lt;/div&gt;').appendTo(buttonContainer);
     // Create the button that will start the tour again
-    var button = $($jsontool.serialize("&lt;button id='tourResume' class='btn btn-default btn-xs'&gt;$services.icon.renderHTML('info') #tr('tour.popover.show')&lt;/button&gt;")).appendTo(buttonContainer);
+    var button = $($jsontool.serialize("&lt;button id='tourResume' class='btn btn-default btn-xs'&gt;$services.icon.renderHTML('info')")
+      + "#tr('tour.popover.show')"
+      + $jsontool.serialize("&lt;/button&gt;"))
+      .appendTo(buttonContainer);
```

Comment:
Of course, in the best scenario this piece of JavaScript would be refactored to get rid of the velocity parsing, which would be greatly beneficial to readability and performance.
In the meantime, what do you think of the solution below?
```suggestion
    const button = $('&lt;button id=\'tourResume\' class=\'btn btn-default btn-xs\'&gt;&lt;/button&gt;')
      .html("$escapetool.javascript($services.icon.renderHTML('info')) #tr('tour.popover.show')")
```
In my option, this makes it clearer what's plain static text, and what's generated from Velocity.